.travis.yml

```diff
@@ -51,15 +51,15 @@ env:
     # does not work with python 3
     - BOTO_CONFIG=/tmp/bogusvalue
   matrix:
+    - TOX_ENV=py27-backend_postgres KUBERNETES_VERSION=v1.8.0
+    - TOX_ENV=py35-backend_postgres KUBERNETES_VERSION=v1.9.0
```

Comment:
Maybe also add Kubernets 1.10?

Reply:
I definitely want to add 1.10, but I wanted to get these fixes in first (basically want to get to a point of "stability" and then add on past that)